install.sh

```diff
@@ -0,0 +1,44 @@
+NAME_PROJECT="jaylenw/AngularJsTodoApp"
+
+clear
+echo " ------------------ $NAME_PROJECT  ------------------ "
+
+echo "Fetches the list of available updates ..."
+sudo apt-get update
```

Comment:
For installations like this, we will use apt instead of apt-get. I will make the changes to this and others.